doc/user_guide/user_guide.md

```diff
@@ -40,32 +40,132 @@ within the scope of the project.
 ```buildoutcfg
 pip install exasol_sagemaker_extension.whl
 ```
-### Install The Pre-built Container 
-- Upload the pre-built container into BucketFS. In order to do that you can use 
-either a [http(s) client](https://docs.exasol.com/database_concepts/bucketfs/file_access.htm) 
-or the [bucketfs-client](https://github.com/exasol/bucketfs-client). 
-The following example uploads a container to BucketFS through curl command, a http(s) client:
-```buildoutcfg
-curl -vX PUT -T \ 
-    "<CONTAINER_FILE>" 
-    "http://w:<BUCKETFS_WRITE_PASS>@$bucketfs_host:<BUCKETFS_PASS>/<BUCKETFS_NAME>/<PATH_IN_BUCKET><CONTAINER_FILE>"
-```
-For more details please check [Adding New Packages to Existing Script Languages](https://docs.exasol.com/database_concepts/udf_scripts/adding_new_packages_script_languages.htm).
+### The Pre-built Language Container
 
-- Activate the uploaded container through adjusting session parameter `SCRIPT_LANGUAGES`. 
-The activating can be performed for either session-wide (`ALTER SESSION`) or 
-system-wide (`ALTER SYSTEM`). The following example query activates the container session-wide:
-```buildoutcfg
-ALTER SESSION SET SCRIPT_LANGUAGES=\
-<ALIAS>=localzmq+protobuf:///<BUCKETFS_NAME>/<BUCKET_NAME>/<PATH_IN_BUCKET><CONTAINER_NAME>/?\
-        lang=<LANGUAGE>#buckets/<BUCKETFS_NAME>/<BUCKET_NAME>/<PATH_IN_BUCKET><CONTAINER_NAME>/\
-        exaudf/exaudfclient_py3
-```
-where `ALIAS` is _PYTHON3_SME_, `LANGUAGE` is _python_ in the sagemaker-extension project.
-
-For more details please check [Adding New Packages to Existing Script Languages](https://docs.exasol.com/database_concepts/udf_scripts/adding_new_packages_script_languages.htm).
+This extension requires the installation of the language container for this 
+extension to run. It can be installed in two ways: Quick and Customized 
+installations
 
+#### Quick Installation
+The language container is downloaded and installed by executing the 
+deployment script below with the desired version. Make sure the version matches with your installed version of the 
```

Comment:
For the quick installation, we should actually use a proper default value for --version. However, this is a topic for a later ticket.